debian/postrm

```diff
@@ -1,3 +1,15 @@
 #!/bin/sh -e
+case "$1" in
+    remove)
+    userdel plight
```

Comment:
I agree.  I did some reading any everyone recommends leaving the users alone. https://fedoraproject.org/wiki/Packaging:UsersAndGroups

My one concern is that upgrading this package will not fix problems with how the user was created by older versions.